docker-compose.yml

```diff
@@ -7,9 +7,7 @@ services:
       - MYSQL_USER=remo
       - MYSQL_PASSWORD=remo
       - MYSQL_ROOT_PASSWORD=root
-    entrypoint:
-      - docker-entrypoint.sh
-      - --max_allowed_packet=2M
+    command: mysqld --max_allowed_packet=64M
```

Comment:
Maybe you are using a old version of mysql image.
You can pull the latest version with ``docker-compose pull``